docs/chapters/sdk/template-service.rst

```diff
@@ -0,0 +1,104 @@
+Template Service
+================
+
+A template service serves as an example CMake_ project for service developers that should be used as a starting point while creating platform services, as well as as a set of guidelines to follow in source repositories. The guidelines described are not set-in-stone. Also, in general consistency is better than getting stuck on difficult-to-apply guidelines so please use these to an extent which is reasonable. The point is to support a common pattern within the project.
```

Comment:
Shouldn't we break lines to a reasonable length? I'm not sure if this is a mistake or if there's some reason for it.